### PR TITLE
first draft of install

### PR DESCRIPTION
**Pull request by TyDunn**

### CODE REVIEW

**Review by adrianbr — APPROVED** [review]
looks good

File: docs/website/docs/installation.mdx [hunk 1]
```diff
@@ -8,4 +8,102 @@ import Tabs from '@theme/Tabs';
 
 ## Set up environment
 
-## Install `dlt` library
\ No newline at end of file
+Make sure you are using **Python 3.8 or 3.9**:
```
Comment (rudolfix) [inline docs/website/docs/installation.mdx:11]: everything > 3.8 should work. the bigquery client does not work on python 3.11 yet but dlt itself runs on it

File: docs/website/docs/installation.mdx [hunk 2]
```diff
@@ -8,4 +8,102 @@ import Tabs from '@theme/Tabs';
...
+python3 --version
+pip3 --version
+```
+
+If not, then please follow the instructions below to install it:
+
+<Tabs values={[{"label": "Ubuntu", "value": "ubuntu"}, {"label": "macOS", "value": "macos"}, {"label": "Windows", "value": "windows"}]}  groupId="operating-systems" defaultValue="ubuntu">
+  <TabItem value="ubuntu">
+
+You can install Python 3.9 with an `apt` command.
```
Comment (rudolfix) [inline docs/website/docs/installation.mdx:23]: we should go for python 3.10 btw. this is actually version on which we test on github

File: docs/website/docs/installation.mdx [hunk 3]
```diff
@@ -8,4 +8,102 @@ import Tabs from '@theme/Tabs';
...
+
+```bash
+brew update
+brew install python@3.9
+```
+
+  </TabItem>
+  <TabItem value="windows">
+
+Once you have installed the [Microsoft VC++ Compiler](https://visualstudio.microsoft.com/visual-cpp-build-tools/), you can install [Python 3.9](https://www.python.org/downloads/windows/) (64-bit version) for Windows.
```
Comment (rudolfix) [inline docs/website/docs/installation.mdx:43]: I must verify this. we should have no dependencies that require compilation. where did you take this info from?

File: docs/website/docs/installation.mdx [hunk 4]
```diff
@@ -8,4 +8,102 @@ import Tabs from '@theme/Tabs';
...
+  </TabItem>
+</Tabs>
+
+Now that you have Python installed, you should create virtual environment for your project:
+
+<Tabs values={[{"label": "Ubuntu", "value": "ubuntu"}, {"label": "macOS", "value": "macos"}, {"label": "Windows", "value": "windows"}]}  groupId="operating-systems" defaultValue="ubuntu">
+  
+  <TabItem value="ubuntu">
+
+Create a new virtual environment by making a `./env` directory to hold it.
```
Comment (rudolfix) [inline docs/website/docs/installation.mdx:58]: ok let's add `.env` to standard git ignore :)

**Review by rudolfix — CHANGES_REQUESTED** [review]

File: docs/website/docs/installation.mdx [hunk 5]
```diff
@@ -8,4 +8,102 @@ import Tabs from '@theme/Tabs';
...
+If not, then please follow the instructions below to install it:
+
+<Tabs values={[{"label": "Ubuntu", "value": "ubuntu"}, {"label": "macOS", "value": "macos"}, {"label": "Windows", "value": "windows"}]}  groupId="operating-systems" defaultValue="ubuntu">
+  <TabItem value="ubuntu">
+
+You can install Python 3.9 with an `apt` command.
+
+```bash
+sudo apt update
+sudo apt install python3.9
```
Comment (rudolfix) [inline docs/website/docs/installation.mdx:27]: I also noticed that `venv` module is not installed automatically so your instructions to create virtual env will fail. on linux
sudo apt install python3.10-venv

is it the same on brew?

**Review by rudolfix — CHANGES_REQUESTED** [review]

File: docs/website/docs/installation.mdx (hunk 3 above)
Comment (TyDunn) [reply docs/website/docs/installation.mdx:43]: Sweet. If don't need this, even better!

**Pull request merged**